StyleCop.Analyzers/StyleCop.Analyzers/Lightup/LightupHelpers.cs

```diff
@@ -65,15 +65,17 @@ internal static bool CanWrapObject(object obj, Type underlyingType)
                 return false;
             }
 
-            ConcurrentDictionary<Type, bool> wrappedObject = SupportedObjectWrappers.GetOrAdd(underlyingType, _ => new ConcurrentDictionary<Type, bool>());
-
             // Avoid creating the delegate if the value already exists
```

Comment:
📝 This comment is sufficient for the method (it already states the complete intent, and it's just a simple bug that we didn't adhere to it)